src/Application.php

```diff
@@ -304,6 +320,10 @@ protected function loadConfig()
         if (isset($config['craft_translations_path'])) {
             define('CRAFT_TRANSLATIONS_PATH', rtrim($config['craft_translations_path'], '/').'/');
         }
+        
+        if (isset($this->vendorPath)) {
```

Comment:
@dnunez24 These lines are going to be reverted and replaced with a `craft_vendor_path` config setting in the next release. The current code breaks in the scenario where you have Craft CLI installed via composer, but you don't have Craft itself installed via composer.